### ISSUE TITLE
MSI UI Icon

### ISSUE THREAD

**Issue by tmontes** [mu-editor/pup#103]
Follow-up from #90.

* The minimal MSI installation progress UI displays a generic icon.
* It would be nice for it to include the given application's icon, if any.

(maybe addressing #91, requiring changes to the install progress UI, helps us understand how to attain this) :)

**Comment by tmontes** [comment]
NOTE:

* While addressing #91 under #105, learned that we will need two bitmaps for the MSI UI.
* Per the WiX docs at https://wixtoolset.org/documentation/manual/v3/wixui/wixui_customizations.html:
  * `WixUIDialogBmp` is a 493×312 pixel bitmap.
  * `WixUIBannerBmp` is a 493×58 pixel bitmap.

IDEA:

* Maybe we could use `pillow` to auto-generate such bitmaps if/when an icon is given.
* How hard can that be? :-)

**Comment by tmontes** [comment]
PROGRESS:

* Played around a bit with `pillow`.
* This mini program produces a simple banner given an ICO file.
* Pimping things up a bit could be nice... (how?)

```
#!/usr/bin/env python3.7

from PIL import Image


BANNER_WIDTH = 493
BANNER_HEIGHT = 312
BANNER_LEFT = 164

BANNER_BACK = 'white'

ICON_WIDTH = 64
ICON_HEIGHT = 64

ICON_X = (BANNER_LEFT - ICON_WIDTH) // 2
ICON_Y = 24


if __name__ == '__main__':

    ico = Image.open('mu.ico')
    ico64 = ico.resize((ICON_WIDTH, ICON_HEIGHT))

    banner = Image.new('RGBA', (BANNER_WIDTH, BANNER_HEIGHT), BANNER_BACK)
    banner.alpha_composite(ico64, (ICON_X, ICON_Y))

    banner.save('banner.png')
```

NEXT:

* Let's try and integrate such a banner into the MSI files, shall we?
*

**Comment by tmontes** [comment]
CLOSING:

* Just merged PR #172.
* One less thing todo under #85.
